<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSort</class>
 <widget class="QDialog" name="EditSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Sort Order</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="field_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="field_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sort_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edit Custom Sort Order&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSort</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSort</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
